nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,12 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 1.0.1
+***************
+
+* Reverted :c:func:`nrf_getaddrinfo()` behavior to be the same as in v0.8.99 due to the Carrier library not being compatible with the newly introduced POSIX errors codes yet.
+* Removed the `nrf_gai_error.h` header.
```

Comment:
```suggestion
* Reverted the :c:func:`nrf_getaddrinfo` function behavior to be the same as in v0.8.99, since the LwM2M carrier library is not compatible with the newly introduced POSIX errors codes yet.
* Removed the :file:`nrf_gai_error.h` header.
```

Reply:
Fixed